packages/dmn-js-drd/src/features/modeling/cmd/UpdatePropertiesHandler.js

```diff
@@ -37,8 +39,8 @@ UpdatePropertiesHandler.$inject = [ 'elementRegistry', 'moddle' ];
  */
 UpdatePropertiesHandler.prototype.execute = function(context) {
 
-  var element = context.element,
-      changed = [ element ];
+  const { element, properties } = context,
```

Comment:
[optional] I'd consider to either have all declarations in ES6 (const, let), or leave this file with `var`s only. Right now it looks weird.